### DISCUSSION TITLE
Dynamically fill the bucket

### DISCUSSION THREAD

**Discussion by ShriramDeshkar** [vladimir-bukhtoyarov/bucket4j#405] [Q&A]
I m currently working on Azure Java function, those functions will be calling some external http restful endpoints.
I want to limit number of request my azure function sending to external endpoints dynamically e.g. If I start with 10 requests per seconds , so my bucket size will be 10 at the start of each second. But if external endpoints starts responding slowly because of some issue ,  I want to change my bucket size to 7 during the runtime.
Can this be possible ? Can I change the bucket size during the runtime ?

**Comment by vladimir-bukhtoyarov** [comment]
https://bucket4j.com/8.3.0/toc.html#configuration-replacement
